CHANGELOG.latest.md

```diff
@@ -1,6 +1,24 @@
-- Temporarily disable ETags
-     https://github.com/RevenueCat/purchases-android/pull/322
-- Converts attribution data to use subscriber attributes
-     https://github.com/RevenueCat/purchases-android/pull/315
-- Clarify usage of sharedInstance
-     https://github.com/RevenueCat/purchases-android/pull/320
+### Identity V3:
+
+#### New methods
+- Introduces `logIn`, a new way of identifying users, which also returns whether a new user has been registered in the system. 
+`logIn` uses a new backend endpoint. 
+- Introduces `logOut`, a replacement for `reset`. 
+
+#### Deprecations / removals
+- removes `createAlias`
+- deprecates `identify` in favor of `logIn`
+- deprecates `reset` in favor of `logOut`
+- deprecates `allowSharingPlayStoreAccount` in favor of dashboard-side configuration
+
+    https://github.com/RevenueCat/purchases-android/pull/250
+    https://github.com/RevenueCat/purchases-android/pull/260
+    https://github.com/RevenueCat/purchases-android/pull/252
+
+
+### Other changes: 
+- Fixed CI issues with creating pull requests
+    https://github.com/RevenueCat/purchases-android/pull/324
+
+### HOLD
+This PR should not be merged. Instead, we'll make a regular release from develop once beta testing is complete. 
```

Comment:
Do we want this? I noticed this section is not in the iOS PR

Reply:
LOL no, I copy/pasted the PR description from iOS into the changelog here 🤦🏻‍♂️